Wip solution seed data

vinnyalfieri opened the pull request:
@jmburges @deniznida @ipc103 

I expanded the solution to show the actual Database data to display on the screen for each view. Also added a "Figures" view to display more data from the Database. 

This introduces passing data from your controller to the actual view via class_variables

ahimmelstoss committed:
update resources section

fislabstest committed:
Add content license

deniznida committed:
Update README.md

AnnJohn committed:
Removed outdated Resource link

spencerarogers committed:
Add contributing instructions

vinnyalfieri committed:
added seed data to solution

vinnyalfieri committed:
fixed solution to work with seed data and added more solution code to expand on the functionality of the lab rather than just passing the tests

vinnyalfieri committed:
shits

vinnyalfieri committed:
Merge branch 'master' into wip-solution-seed-data

jmburges commented on app/views/figures/index.erb:
```diff
@@ -0,0 +1,10 @@
+<table>
+  <th>Person Name</th>
+  <tr>
+    <% @figure.each do |figures| %>
```
@figure is your variable name and `figures` is the block variable? I you sure it shouldn't be `@figures` is the instance variable and `figure` is the block variable?

jmburges commented on app/views/landmarks/index.erb:
```diff
@@ -0,0 +1,14 @@
+  <th>Name</th>
+  <th>Year Completed</th>
+  <tr>
+    <% @landmark.each do |land| %>
```
shouldn't that be the `@landmarks`? It's multiple landmarks right?

jmburges replied:
also `land` as a block variable doesn't make sense. It's not all of a sudden a piece of land right? isn't is `landmark`?

jmburges commented:
back at you

vinnyalfieri committed:
updated seed data to remove unused elements

vinnyalfieri committed:
updated landmark to landmarks so we stick to naming conventions

vinnyalfieri commented:
Updated the seed data to remove unused elements.
Updated the Landmarks controller to use proper naming convention. Pluralized @landmark since it is supposed to hold MANY landmarks.

@jmburges @AnnJohn closes issue #6
